### PR TITLE
Feature/add migrations

### PR DESCRIPTION
**Pull request by MatthewEllisTx**
Adds service_types and service_type_categories table migrations following this plan

https://app.dbdesigner.net/designer/schema/0-schema-w-json-family-promise-labs-35

### CODE REVIEW

**Comment by MatthewEllisTx** [comment]
Description
What work was done?
• created new migration files
Why was this work done?
• In order to restructure the database to fit our desired schema
Type of change
• Breaking feature? We will have to rewrite some of the model files
Change Status
• Completed, ready to review and merge
Checklist
• My code follows the style guidelines of this project
• I have performed a self-review of my own code
• My changes generate no new warnings
• There are no merge conflicts

File: data/newMigrations/006-service_types.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('service_types', (tbl) => {
+    tbl.increments('service_type_id').primary();
+    tbl.foreign('program_id').references('programs.program_id');
+    tbl.string('service_type_name', 255);
+    tbl.text('service_type_description');
+    tbl.jsonb('service_type_entry_model');
+    table.timestamps(true, true);
```
Comment (samplesn08) [inline data/newMigrations/006-service_types.js:8]: This should be "tbl.timestamps" rather than "table.timestamps"

**Review by Monicascz — APPROVED** [review]
Nicely done! Great job naming your PR and detailing what changes you made.

File: data/newMigrations/006-service_types.js (hunk 1 above)
Comment (MatthewEllisTx) [reply data/newMigrations/006-service_types.js:8]: Ah, you're right

**Pull request merged**